low-level/files/opt/seagate/sspl/setup/sspl_post_install.py

```diff
@@ -135,15 +131,23 @@ def validate_dependencies(setup):
                 "python3": "3.6.8",
                 "smartmontools": "7.0",
                 "systemd-python36": "1.0.0",
-                "udisks2": "2.8.4"
+                "udisks2": "2.8.4",
+                "python36-dbus": "1.2.4",
+                "python36-gobject": "3.22.0",
+                "python36-paramiko": "2.1.1",
+                "python36-psutil": "5.6.7",
             },
             "el8": {
                 "hdparm": "9.54",
                 "lshw": "B.02.19.2",
                 "python36": "3.6.8",
                 "smartmontools": "7.1",
                 "python3-systemd": "234",
-                "udisks2": "2.9.0"
+                "udisks2": "2.9.0",
+                "python3-dbus": "1.2.4",
+                "python3-gobject": "3.28.3",
+                "python3-paramiko": "2.4.3",
+                "python3-psutil": "5.4.3",
```

Comment:
Please make sure the package and its version are align with RE requirements.

Reply:
RE is not ready yet, this time every component just have to register their requirements here 
https://seagate-systems.atlassian.net/wiki/spaces/PRIVATECOR/pages/579076097/F-63D+Rocky+Linux+8.4+Support+PoC

Reply:
Yes, @cdeshmukh , EL8 repos has mostly only the latest version of every rpm available, no previous versions are available. so all pkgs are latest available on EL8 repositories.

Reply:
Got it